modules/rhcos-environment-variables.adoc

```diff
@@ -8,6 +8,8 @@
 [role="_abstract"]
 You can use the following environment variables to configure {op-system} node scanning on {product-title-short}.
 
+//needs updating for 4.6 with new variables?
```

Comment:
These are shared between the two configurations for now. The only difference currently is the new toggle for Node Indexing on Compliance:

```
|ROX_NODE_INDEX_ENABLED
|Controls whether Node Indexing is enabled for this cluster. The default value is `false`.
```

Reply:
should we also make it clear indexing = Scanner V4-based?